cmd/deck/pluginhelp.go

```diff
@@ -52,6 +59,27 @@ func (ha *helpAgent) getHelp() (*pluginhelp.Help, error) {
 		return ha.help, nil
 	}
 
+	//Setting Root CAs if SSL is enabled
+	hookPath, err := url.Parse(ha.path)
+	if err != nil {
+		return nil, fmt.Errorf("error parsing hook path: %w", err)
+	}
+	if hookPath.Scheme == tlsEnabledScehma {
+		caCert, err := os.ReadFile(ha.cert)
+		if err != nil {
+			return nil, fmt.Errorf("error decoding cert file: %w", err)
+		}
+		caCertPool := x509.NewCertPool()
+		caCertPool.AppendCertsFromPEM(caCert)
+		//Error is eaten here for testing purposes. There should be no reason the transport is not of type HTTP
+		if transport, ok := http.DefaultTransport.(*http.Transport); ok {
+			transport.TLSClientConfig = &tls.Config{
+				RootCAs: caCertPool,
+			}
+			http.DefaultTransport = transport
```

Comment:
`helpAgent` should get a custom client member with this transport instead of modifying the *global* default transport (this would affect anything else in hook that is a http client and relies on the default)